tobler/util/util.py

```diff
@@ -139,7 +142,10 @@ def h3fy(source, resolution=6, clip=False, buffer=False, return_geoms=True):
         else:
             source = source.to_crs(4326)
 
-    source_unary = shapely.force_2d(source.unary_union)
+    if GPD_10:
+        source_unary = shapely.force_2d(source.union_all())
```

Comment:
I future, we may want to be able to pass kwargs here https://github.com/geopandas/geopandas/pull/3151